### ISSUE TITLE
Gradle Plugin: Support configuration cache

### ISSUE THREAD

**Issue by nbrugger-tgm** [casid/jte#204]
Starting with Gradle 8.1 the configuration-cache is not unstable anymore 🥳 

At the moment the jte gradle plugin completely halts the gradle process when the config cache is enabled

```
* What went wrong:
Execution failed for task ':core:generateJte'.
> Cannot query the value of this property because it has no value available.
```

```
org.gradle.unsafe.configuration-cache=true
```

**Comment by edward3h** [comment]
8.1 is a release candidate at the moment? Thanks for the heads up.

I'm kinda busy this week but can try and find time to look into it next week.

**Comment by nbrugger-tgm** [comment]
Yes it is RC right now so its not too pressing, thank you for taking a look into it 👍🏻

**Comment by edward3h** [comment]
@casid Do you have any preference for which Gradle versions to support? I would probably try and keep backwards compatibility with 7.x, though I know there were some discussions about Kotlin version support that were related.

**Comment by casid** [comment]
@edward3h just came back from vacation, so sorry for the late reply. You're right, for Kotlin/Gradle the minimum Gradle version is now 8.x. Other combinations currently still work with Gradle < 8.x, so it's nice that you added support :-)

Also, thanks for the PR, I'd have no idea what to do here!

**Comment by nbrugger-tgm** [comment]
Is the feature already released? If yes this ticket could be closed with a version note.

Thank you for your effort

**Comment by casid** [comment]
@nbrugger-tgm I just published [version 2.3.1](https://github.com/casid/jte/releases/tag/2.3.1), that includes this feature.

Thank you @edward3h, for making this happen :-)
